docs/how-to-guides/hello-world-example.md

```diff
@@ -7,131 +7,120 @@ import TabItem from '@theme/TabItem';
+Rancher Desktop works with two container engines, [containerd](https://containerd.io/) and [Moby](https://mobyproject.org/), the open-sourced components of the Docker ecosystem. For `nerdctl`, use the **containerd** runtime. For `docker`, use the **dockerd(moby)** runtime.
 
-### Hello World example
+### Example#1 - Build Image & Run Container
 
-**Create a folder:**
+#### Create a folder
 ```
 mkdir ../hello-world
 cd ../hello-world
 ```
 
-**Create a Dockerfile with the command below.**
+#### Create a Dockerfile with the command below
 ```
 FROM alpine  
 CMD ["echo", "Hello World!!"]
 ```
 
-**Build and run the image for verification purposes:**
+#### Build and run the image for verification purposes
 
 <Tabs groupId="container-runtime">
   <TabItem value="nerdctl" default>
 
 ```
-nerdctl build -t helloworld:v1.0 .
+nerdctl build --tag helloworld:v1.0 .
 nerdctl images | grep helloworld
 nerdctl run --rm helloworld:v1.0
-nerdctl rmi helloworld:v1.0 #To remove the image
+#To remove the image
+nerdctl rmi helloworld:v1.0 
 ```
 
   </TabItem>
   <TabItem value="docker">
 
 ```
-docker build -t helloworld:v1.0 .
+docker build --tag helloworld:v1.0 .
 docker images | grep helloworld
 docker run --rm helloworld:v1.0
-docker rmi helloworld:v1.0 #To remove the image
+#To remove the image
+docker rmi helloworld:v1.0 
 ```
 
   </TabItem>
 </Tabs>
 
-### NGINX example
+### Example#2 - Build Image & Deploy Container to Kubernetes
 
-Make sure that you switch the `container runtime` setting in the **Kubernetes Settings** panel to either `dockerd` or `containerd` as needed.
+Make sure that you switch the **Container Runtime** setting in the **Kubernetes Settings** panel to either `dockerd` or `containerd` as needed.
 
-**Create a folder and add a sample index.html file as follows:**
+#### Create a folder and add a sample index.html file as follows
 ```
 mkdir ../nginx
 cd ../nginx
 echo "<h1>Hello World from NGINX!!</h1>" > index.html
 ```
 
-**Create a Dockerfile with the command below.**
+#### Create a Dockerfile with the command below
 ```
 FROM nginx:alpine
 COPY . /usr/share/nginx/html
 ```
 
-**Build and run the image for verification purposes:**
+#### Build Image from code locally
+
+:warning: **Note:** Please note that you need to pass the flag `--namespace k8s.io` to the `nerdctl` build command, so that `nerdctl` builds the image and then makes it available in the `k8s.io` namespace.
 
 <Tabs groupId="container-runtime">
   <TabItem value="nerdctl" default>
 
 ```
-nerdctl build -t nginx-helloworld:v1.0 .
-nerdctl images | grep nginx-helloworld
-nerdctl run -d -p 8080:80 --name my-site nginx-helloworld:v1.0
+nerdctl --namespace k8s.io build --tag nginx-helloworld:latest .
+nerdctl images --namespace k8s.io | grep nginx-helloworld
```

Comment:
```suggestion
nerdctl --namespace k8s.io images | grep nginx-helloworld
```

Reply:
I missed to fix to this in my earlier commit. Anyways, fixed it now.